scripts/ci/mlflow-seldon-e2e.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
```

Comment:
That script is 90% similar to `mlflow-e2e.sh`. I think you can make some changes to `mlflow-e2e.sh` so it can also support running the seldon-core tests depending on the value set for `WORKFLOW`.

Reply:
good point